exporter/prometheusremotewriteexporter/exporter_test.go

```diff
@@ -457,20 +475,23 @@ func Test_PushMetrics(t *testing.T) {
 	}
 
 	tests := []struct {
-		name               string
-		metrics            pmetric.Metrics
-		reqTestFunc        func(t *testing.T, r *http.Request, expected int, isStaleMarker bool)
-		expectedTimeSeries int
-		httpResponseCode   int
-		returnErr          bool
-		isStaleMarker      bool
-		skipForWAL         bool
+		name                 string
+		metrics              pmetric.Metrics
+		reqTestFunc          func(t *testing.T, r *http.Request, expected int, isStaleMarker bool)
+		expectedTimeSeries   int
+		httpResponseCode     int
+		returnErr            bool
+		isStaleMarker        bool
+		skipForWAL           bool
+		hasFailedTranslation bool
```

Comment:
```suggestion
		expectedFailedTranslations int
```
Switching to this will simplify your test runner code a bit.